Documentation/app-operations.md

```diff
@@ -0,0 +1,197 @@
+This operation is idempotent.
+
+```bash
+rkt app start <pod-uuid> --app=<app-name> <arguments>
+```
+
+## rkt app stop
+Stops a running application gracefully.
+
+```bash
+rkt app stop <pod-uuid> --app=<app-name> <arguments>
+```
+
+## rkt app list
+Lists the applications that are inside a pod, running or stopped.
+
+```bash
+rkt app list <pod-uuid> <arguments>
+```
+
+## rkt app status
+Returns the execution status of application inside a pod.
+
+```bash
+rkt app status <pod-uuid> --app=<app-name> <arguments>
+```
+
+which returns status information for an example mysql application along the lines of:
+
+```bash
+● mysql.service - Application=mysql Image=registry-1.docker.io/library/mysql
+   Loaded: loaded (/usr/lib64/systemd/system/mysql.service; static; vendor preset: enabled)
+      Active: active (running) since Wed 2016-07-09 17:00:50 UTC; 8min 38s ago
+       Main PID: 6 (mysqld)
+          CGroup: /machine.slice/machine-rkt\x2d11efeefb\x2d2e8d\x2d47d0\x2da4ee\x2de1631327acac.scope/system.slice/mysql.service
+                     └─6 mysqld
+
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.047413Z 0 [Note]   - '::' resolves to '::';
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.047669Z 0 [Note] Server socket created on IP: '::'.
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.048279Z 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.080176Z 0 [Warning] 'db' entry 'sys mysql.sys@localhost' ignored in --skip-name-resolve mode.
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.085072Z 0 [Warning] 'proxies_priv' entry '@ root@localhost' ignored in --skip-name-resolve mode.
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.093485Z 0 [Note] InnoDB: Buffer pool(s) load completed at 160713 17:01:03
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.095603Z 0 [Warning] 'tables_priv' entry 'sys_config mysql.sys@localhost' ignored in --skip-name-resolve mode.
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.111731Z 0 [Note] Event Scheduler: Loaded 0 events
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.113195Z 0 [Note] mysqld: ready for connections.
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: Version: '5.7.13'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  MySQL Community Server (GPL)
+```
+
+## rkt app exec
+Executes a command inside an application.
```

Comment:
What does inside an application mean? (Can probably steal language from `rkt exec` for this - since AIUI this should be semantically identical)